src/databricks/labs/lsql/dashboards.py

```diff
@@ -193,18 +194,47 @@ def split(self) -> tuple[str, str]:
         return "", self._content
 
 
+class FilterHandler(BaseHandler):
+    """Handle filter files."""
+
+    def _parse_header(self, header: str) -> dict:
+        if not header:
+            return {}
+        metadata = json.loads(header)
+        # The user can either provide a single filter column as a string or a list of filter columns
+        # Only one of column or columns should be set
+        filter_col = metadata.pop("column", None)
+        filter_cols = metadata.pop("columns", None)
+        if not filter_col and not filter_cols:
+            raise ValueError(f"Neither column nor columns set in {self._path}")
+        if filter_col and filter_cols:
+            raise ValueError(f"Both column and columns set in {self._path}")
+        # If a single column is provided, convert it to a list of one column
+        metadata["filters"] = [filter_col] if filter_col else filter_cols
+        metadata["widget_type"] = WidgetType(metadata.pop("type", "MULTI_SELECT").upper())
```

Comment:
```suggestion
        metadata["widget_type"] = WidgetType(metadata.pop("type", "DROPDOWN").upper())
```
dropdown is considered to be a default filter, as UX has way fewer clicks to filter a dashboard

Reply:
@nfx per current lsql version, the default is MultiSelectSpec and existing UCX dashboards relied on that. Do you want to change the default?